ntc_templates/templates/alcatel_aos_show_lldp_remote-system.textfsm

```diff
@@ -0,0 +1,49 @@
+Value Filldown PORT ((\d+\/?)+)
```

Comment:
I know ... we worked up a clever regex in past template PRs, but I don't see the maintainers wanting to use that.

Especially since we/I learned about `A/1` and `esat-1/1/1` type ports.

```suggestion
Value Filldown PORT (\S+)
```

Reply:
yeah... maybe it's a better solution the `\S+` although i liked more the one you came up with ahahahaha